phpunit.xml

```diff
@@ -3,7 +3,6 @@
         xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
         bootstrap="tests/bootstrap.php"
         colors="true"
-        stopOnFailure="true"
```

Comment:
This should make testing faster, as all the errors are shown. Picking low handing fruit is easier than to be forced to solve first failed case.